src/main/java/com/google/finapp/SpannerDaoInterface.java

```diff
@@ -0,0 +1,26 @@
+/*
+description goes here...
+ */
+
+package com.google.finapp;
+
+import com.google.cloud.ByteArray;
+import com.google.cloud.spanner.*;
+
+import java.math.BigDecimal;
+
+public interface SpannerDaoInterface {
```

Comment:
At some point, we'd want to add javadocs for these classes and methods. It could be a good practice if you want to take that up now.